Assets/MixedRealityToolkit/Definitions/InputSystem/MixedRealityInputSystemProfile.cs

```diff
@@ -56,6 +56,16 @@ public SystemType RaycastProviderType
             internal set { raycastProviderType = value; }
         }
 
+        /// <summary>
+        /// Maximum number of colliders that can be detected in a SphereOverlap scene query.
+        /// </summary>
+        public int FocusQueryBufferSize => focusQueryBufferSize;
+
+        [SerializeField]
+        [Range(32, 2048)]
```

Comment:
Shouldn't this just be 1? i.e. why can't I try to detect just a single object?

Reply:
Good point.